VideoLocker/src/main/java/org/edx/mobile/module/db/impl/DbHelper.java

```diff
@@ -109,12 +116,60 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 createAssessmentTable(db);
             }
 
+            if (oldVersion < 6) {
+                Cursor cursor = db.query(false, DbStructure.Table.DOWNLOADS,
+                        new String[]{DbStructure.Column.ID, DbStructure.Column.USERNAME,
+                                DbStructure.Column.FILEPATH}, null, null, null, null, null, null);
+                if (cursor != null) {
+                    final File appExternalDir = mContext.getExternalFilesDir(null).getParentFile();
+                    try {
+                        while (cursor.moveToNext()) {
+                            final int id = cursor.getInt(0);
+                            final String username = cursor.getString(1);
+                            final String filePath = cursor.getString(2);
+                            final String encryptedUsername = Sha1Util.SHA1(username);
+                            final String newFilePath = filePath.replace(
+                                    appExternalDir.getAbsolutePath() + "/" + username,
+                                    appExternalDir.getAbsolutePath() + "/"
+                                            + AppConstants.Folders.VIDEOS + "/"
+                                            + encryptedUsername);
```

Comment:
Also, do we know that the `getExternalFilesDir()` method would return the same media path that was constructed manually previously via the global `Environment.getExternalStorageDirectory()` method? The documentation on the former states that the returned path may change over time if different shared storage media is inserted, but the latter's documentation only states that it's path may change between different platform versions. If there may be a discrepancy, then we should check for the previous path and replace it with the latter (possibly via regex).